modules/admin_manual/pages/installation/apps_management_installation.adoc

```diff
@@ -5,105 +5,104 @@
 
 After installing ownCloud, you may provide added functionality by installing applications.
 
-== Supported Apps
+== Installing and Managing Apps
 
-See xref:installation/apps_supported.adoc[supported apps] for a list of supported Enterprise edition apps.
+=== Installing Apps Via the ownCloud Marketplace
 
-== Viewing Enabled Apps
+image:installation/top_menu_showing_market_app.png[ownCloud top navigation menu showing the Market app]
 
-During the ownCloud installation, some apps are installed and enabled by
-default, and some are able to be installed and enabled later on. To see
-the status of your installation’s applications, go to your Apps page.
+To add an app, use the _Market_ app, which is accessible from the top-level navigation bar, on the left-hand side of the page. 
+Once in the Market app, click an app's name to view more details about it.
+Once you have done this, you can also install it by clicking btn:[Install]. 
+Clicking _Install_, downloads it from the ownCloud Marketplace, installs, and enables it.
 
-image:oc_admin_apps.png[Apps page for enabling and disabling apps.]
+Sometimes the installation of a third-party app fails silently, possibly because `appcodechecker' => true,` is enabled in `config.php`. 
+When `appcodechecker` is enabled, it checks if third-party apps are using the private API, rather than the public API. 
+If they are, they are not installed.
 
-There, you will see which apps are currently: _enabled_, _not enabled_,
-and _recommended_. You’ll also see additional filters, such as
-Multimedia, Productivity, and Tool for finding more apps quickly.
+NOTE: If you would like to create or add (your own) ownCloud app, please refer to the xref:developer_manual:app/introduction.adoc[developer manual].
 
-== Managing Apps
+=== Installing Apps Manually
 
-In the Apps page, you can enable or disable applications. Some apps have
-configurable options on the Apps page, such as *Enable only for specific
-groups*, but mainly they are enabled or disabled here and are configured
-on your ownCloud _Admin page_, _Personal page_, or in `config.php`.
+To install an app manually instead of by using the Market app, copy the app either into your ownCloud installation's default app folder (`</path/to/owncloud>/apps`) or, ideally, xref:using-custom-app-directories[a custom app directory].
 
-== Adding Apps
+Once the app folder has been copied into place:
 
-Click the app name to view a description of the app and any of the app
-settings in the Application View field. Clicking the *Install* button
-installs the app. If the app is not part of your ownCloud installation,
-it will be downloaded from the ownCloud Marketplace, installed, and
-enabled.
+. Run `composer install` inside the app’s directory.
+. Enable the app via either:
+** Navigating to menu:Settings[Admin > Apps] and clicking btn:[Enable]; or 
+** Using xref:configuration/server/occ_command.adoc#app-command[the occ app command].
```

Comment:
In the _admin manual_, "installing an app manually" means, using a released `*.tar` file. Everything `composer install` is only relevant in the _developer manual_.

Reply:
Thanks for the clarification.

Reply:
The text has been updated.